invoke-ai/InvokeAI
# Prevent caching to ensure updated UI is always shown

## What type of PR is this? (check all applicable)



## Have you discussed this change with the InvokeAI team?

      
## Have you updated all relevant documentation?
- [X] No docs on this 


## Description
Prevent browsers from caching Invoke so that users never see an outdated UI build

## Related Tickets & Documents



- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings


- build the UI
- open it with 9090 (not yarn dev on 5173)
- change the <title> tag in index.html
- rebuild
- refresh the page
- check if the title tag changed
- ## Added/updated tests?

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?
